skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +474,21 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        rpix, phipix = get_r_phi(shape, origin)
```

Comment:
hmm, seems like it should have been this for it to work:
```python
        rpix = radial_grid((origin[1], origin[0]), (shape[1], shape[0]))
        phipix = angle_grid((origin[1], origin[0]), (shape[1], shape[0]))
        phipix = phipix.T
        rpix = rpix.T
```
But you may ignore this and please read the suggestion for using `angle_grid` and `radial_grid` in the test.
I think the trick is that this equivalence is true:
```python
y = np.arange(10)
x = np.arange(5)

Y_xy, X_xy = np.meshgrid(y,x,indexing='xy')
X_ij, Y_ij = np.meshgrid(x,y,indexing='ij')
```
output of `X_ij`, `X_xy`:
```python
In [22]: X_ij
Out[22]: 
array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [2, 2, 2, 2, 2, 2, 2, 2, 2, 2],
       [3, 3, 3, 3, 3, 3, 3, 3, 3, 3],
       [4, 4, 4, 4, 4, 4, 4, 4, 4, 4]])

In [23]: X_xy
Out[23]: 
array([[0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [2, 2, 2, 2, 2, 2, 2, 2, 2, 2],
       [3, 3, 3, 3, 3, 3, 3, 3, 3, 3],
       [4, 4, 4, 4, 4, 4, 4, 4, 4, 4]])
```